test/celeritas/em/MuBremsstrahlung.test.cc

```diff
@@ -115,12 +139,12 @@ TEST_F(MuBremsstrahlungInteractorTest, basic)
     EXPECT_EQ(num_samples, this->secondary_allocator().get().size());
 
     // Note: these are "gold" values based on the host RNG.
-    double const expected_energy[] = {
-        1012.99606184083, 1029.80705246907, 1010.52595539471, 1010.77666768483};
-    double const expected_costheta[] = {0.968418002240112,
-                                        0.999212413725981,
-                                        0.998550042495312,
-                                        0.983614606590488};
+    static double const expected_energy[] = {0.0065836962047077,
+                                             0.072737465588407,
+                                             0.0046101810148469,
+                                             0.0047801585825437};
```

Comment:
Nice fix! Good to see reasonable brem energy here. Now, these seems to be relatively small (expecting that the mean brem energy is roughly a couple of percent of the incident muon kinetic energy. It could be simply a statistical effect (i.e., following a steep falling spectrum) as only 4 interactions are sampled here or the gamma cutoff (1keV here) may be too small.  We will do a separate physics validation anyway, but can you sample something like 10 interactions here to see whether there is any brem energy above MeV?

Reply:
I had the same concern, and when I increased the number of samples to see if it was a statistical effect I did see some larger secondary energies (above 1 MeV). I added the average energy check in the stress test with many more samples so we have an idea what the average secondary energy is for different incident energies.